cognite/neat/v0/session/_read.py

```diff
@@ -826,20 +830,29 @@ def ontology(self, io: Any) -> IssueList:
         importer = importers.OWLImporter.from_file(reader.materialize_path(), source_name=f"file {reader!s}")
         return self._state.data_model_import(importer)
 
-    def instances(self, io: Any) -> IssueList:
+    def instances(self, io: Any, named_graph: str | None = None) -> IssueList:
         self._state._raise_exception_if_condition_not_met(
             "Read RDF Instances",
             empty_data_model_store_required=True,
         )
         reader = NeatReader.create(io)
-        self._state.instances.store.write(extractors.RdfFileExtractor(reader.materialize_path()))
+
+        # validate and convert named_graph to URI
+        named_graph_uri: URIRef | None = None
+        if named_graph:
+            if not re.match(SPACE_COMPLIANCE_REGEX, named_graph):
+                raise NeatValueError(f"Named graph '{named_graph}' does not comply with naming requirements. ")
```

Comment:
nit: Redundant whitespace at the end.
```suggestion
                raise NeatValueError(f"Named graph '{named_graph}' does not comply with naming requirements.")
```